README.md

```diff
@@ -1,4 +1,27 @@
 # webcam-kaleidoscope
+## What is it?
+A party activity, a moving wallpapar or a simple screensaver... We're not sure.
+### What's going on?
+Web camera observing something. 
+May be a table with small things, your face or fingers, or what you'd like to show.
+The application real-time transforms this image like a real kaleidoscope and shows results on the screen. 
+It can rotate it fast or slow, change colors or make it quake, depending on used shaders.
+Used shaders are ruled with a keyboard.
+One can use a standard keyboard and mouse or some mini-keyboard or numpad. 
+We used this one: https://aliexpress.ru/item/1005006109748866.html 
+For others it may be necessary to develop its own controls module (see `src/controls`).
+## What do you need?
+Some *nix system, web camera and screen.
```

Comment:
I'm afraid some system with v4l (are any other *nix systems than Linux support it?)